code-challenges.test.js

```diff
@@ -15,35 +18,184 @@
+            }
+            // const twoArray = (order) => {
+            //     let newArray = []
+            //     let thisArray = order.sort((a, b) => a - b)
+            //     return thisArray.filter((i) => {
+            //         if(thisArray.indexOf[i] > thisArray.indexOf[0] || thisArray.indexOf[i] < thisArray[thisArray.length -1] ){
+            //             newArray.push(thisArray)
+            //         }
+            //         return newArray
+            //       }) 
+            // }
+
+
+            // const twoArray = (order) => {
+            //     let emptyArray = []
+            //     for(let i =0; i <= order.length; i++){
+            //         if(order.indexOf[i] > -8 || order.indexOf[i] < 90 ){
+            //             let thisArray = order.sort((a, b) => a - b)
+            //             emptyArray.push(thisArray)
+            //         }
+            //     } return emptyArray
+            // }
+
+
+
+            // this is what I keep getting 
+            // - Array [
+            //     -   -8,
+            //     -   90,
+            //     - ]
+            //     + Array [] I can't find why I keep getting this extra array?
 
 // --------------------3) Create a function that takes in two arrays as arguments and returns one array with no duplicate values. STRETCH: Use the spread operator to pass in a dynamic number of arguments.
 
 // a) Create a test with an expect statement using the variables provided.
 
-const testArray1 = [3, 7, 10, 5, 4, 3, 3]
-const testArray2 = [7, 8, 2, 3, 1, 5, 4]
+            // create a describe it test that will take two arrays as arguments and only one will be return one array with non duplicate values.
+    describe("oneArray", () => {
+        it("takes in two arrays as arguments and returns one array with no duplicate values", () => {
+            const testArray1 = [3, 7, 10, 5, 4, 3, 3]
+            const testArray2 = [7, 8, 2, 3, 1, 5, 4]
+            expect(oneArray(this)).toEqual([3, 7, 10, 5, 4, 8, 2, 1])
+        })
+    })
+
+    // I failed
+    // ● oneArray › takes in two arrays as arguments and returns one array with no duplicate values
+
+    // ReferenceError: oneArray is not defined
+
+// const testArray1 = [3, 7, 10, 5, 4, 3, 3]
+// const testArray2 = [7, 8, 2, 3, 1, 5, 4]
 // Expected output: [3, 7, 10, 5, 4, 8, 2, 1]
 
 
 // b) Create the function that makes the test pass.
+
+    // create ac function called oneArray
+    //  function will take in two arrays with "this"
+    //create a for loop to iterate through each array
+    // if any index matches any other index, it will not be output
+    // concat the remaining values
+
+        // const oneArray = (one) => {
+        // let thisArray = one.concat()
+        //      for(let i = 0; i < one.length; i++){
+            // if(thisArray[i]=== thisArray[i]){
+            //      thisArray -= 1
+            // }
+        //     } return thisArray
+        // }
+        const oneArray = (one) => {
```

Comment:
You have some good components here. Just have to use them appropriately. Concat the arrays, run the new Set, and then return that output.